alacritty/src/input.rs

```diff
@@ -87,42 +90,59 @@ pub trait ActionContext<T: EventListener> {
 
 #[derive(Debug, Default, Copy, Clone)]
 pub struct Modifiers {
-    mods: ModifiersState,
+    glutin_state: ModifiersState,
```

Comment:
I'm not a fan of `glutin_state`, maybe `glutin_mods_state`?

Reply:
I think just `glutin_mods` should be superior to both here. It's shorter and still conveys the same message.

Reply:
I'm fine with anything that emphasize `glutin` & `mods` in the same time. `state` was super generic and confusing.

Reply:
Completely agree, thanks.